### DISCUSSION TITLE
Github Copilot tends to give me more comments then code

### DISCUSSION THREAD

**Discussion by Titanium-ctrl** [community/community#7603] [Copilot Conversations]
Language: Go

In my experience of using Copilot so far, I have seen that it tends to predict more comments about the flow of my program (I put comments to separate each stage so I know what I am doing as I have decomposed a problem into smaller sub-problems) rather than actual code. In Golang, Copilot tends to only provide code as I am typing it, which is pretty much like the auto-fill/IntelliSense already provided by VSC and other IDEs. Other than that, it predicts comments, and then at times, repeats comments already written.

**Comment by 3sv1n** [comment]
Works fine for me, Android Studio 2021.2.1, WebStorm, PyCharm...
Maybe try refreshing/updating/reinstalling the plugin?
or try updating the IDE.
Hope it helps
